bootstrap/kubeadm/main.go

```diff
@@ -85,6 +86,7 @@ var (
 	webhookCertDir              string
 	healthAddr                  string
 	tokenTTL                    time.Duration
+	tlsOptions                  = flags.TLSOptions{}
```

Comment:
Nit:
should we provide a NewTLSOption method (similar to NewOptions) so we can eventually handle some defaulting in future

Reply:
As of now since we aren't defaulting anything, I'd say it should not matter. Maybe lets hold the change until we introduce some sort of defaulting logic in there.